### PR TITLE
gpu-next: add support for --dither-depth=auto

### PR DESCRIPTION
**Pull request by kasper93**

### CODE REVIEW

File: video/out/d3d11/context.c [hunk 1]
```diff
@@ -154,6 +156,27 @@ static bool d3d11_reconfig(struct ra_ctx *ctx)
 static int d3d11_color_depth(struct ra_swapchain *sw)
 {
     struct priv *p = sw->priv;
+    IDXGIOutput *output = NULL;
```
[the remark this answers is not in the document]
Comment (kasper93) [reply video/out/d3d11/context.c:159]: I've seen that, but it does more fluff, I guess I can factor out DXGI_OUTPUT_DESC1 getter.

[the remark this answers is not in the document]
Comment (kasper93) [reply video/out/d3d11/context.c]: done

**Comment by kasper93** [comment]
It does fix the issue for d3d11, limitations of other APIs are here and would need other way to query to dither target value.

**Review by sfan5 — APPROVED** [review]
lgtm

**Pull request merged**